ci: add issue templates

romanrostislavovich opened the pull request:
Added following issues templates:
- bug_report
- custom
- feature_request

Part of #29

xferra reviewed (changes requested):
Please rebase and use a proper branch name next time.

romanrostislavovich committed:
ci: add issue templates

Added following issues templates:
- bug_report
- custom
- feature_request

Part of #29

xferra commented on .github/ISSUE_TEMPLATE/Bug_report.md:
```diff
@@ -0,0 +1,40 @@
+
+Please complete the following information):
+
+- Version of @linterhub/schema [e.g. 1.0.0]
```
I plan to replace it with more generic text (especially without project name specified), anyway will do it later in scope of separate PR.

xferra reviewed (approved):
Looks good to me
